### PR TITLE
CodeStyle: adopt the Linux CodeStyle

### PR DESCRIPTION
**Pull request by ordex**
This change has been created by using the command:

`indent -linux -psl *.c *.h`

Some minor fixes have then been applied manually to fix
defines.

Signed-off-by: Antonio Quartulli <a@unstable.cc>

### CODE REVIEW

**Comment by ordex** [comment]
Parenthesis around return expressions have also been manually removed.

File: src/text.c [hunk 1]
```diff
@@ -58,37 +58,36 @@ int run = 1;
...
-	{ "verbose",		0,	NULL,	'v' },
-	{ "help",		0,	NULL,	'h' },
-	{ "version",		0,	NULL,	'V' },
-	{ "alternate",		0,	NULL,	'a' },
-	{ "insecure", 		0,	NULL,	'k' },
-	{ "no-clobber",		0,	NULL,	'c' },
-	{ "header",		1,	NULL,	'H' },
-	{ "user-agent",		1,	NULL,	'U' },
-	{ NULL,			0,	NULL,	0 }
+static struct option axel_options[] = {
```
Comment (ismaell) [inline src/text.c]: Alignment of the initializer is now broken.

File: src/text.c [hunk 2]
```diff
@@ -209,503 +205,487 @@ int main( int argc, char *argv[] )
...
-		{
-			printf( _("%i usable servers found, will use these URLs:\n"), j );
-			j = min( j, conf->search_top );
-			printf( "%-60s %15s\n", "URL", _("Speed") );
-			for( i = 0; i < j; i ++ )
-				printf( "%-70.70s %5i\n", search[i].url, search[i].speed );
-			printf( "\n" );
+		search_sortlist(search, i);
+		if (conf->verbose) {
+			printf(_
```
Comment (ismaell) [inline src/text.c]: This line is broken.

**Comment by ismaell** [comment]
@ordex I'm ambivalent to it, but just as a note: `indent -linux` implies `-bbo`, it's intentional.

**Comment by ismaell** [comment]
Check:
* src/axel.c:432

It also broke the MATCH + KEY statements.

**Comment by ordex** [comment]
Actually I rearranged that on purpose manually, but it makes sense to keep the same style as before.
fixed!

**Comment by ismaell** [comment]
Merged.

**Pull request closed without merging**